### PR TITLE
feat(ds): Detect health check routes with trailing slash

### LINKED ISSUE #81028: Trailing slash on Health Check glob patterns
### Problem Statement

The current list of glob patterns for ignoring health check requests do not match URLs with trailing slashes, i.e. `https://mycoolapp.io/ping/` will not be ignored.

I work a lot with Django and the default behavior is to require trailing slashes on URL. Missing the slash will auto-redirect to the same URL with the trailing slash and disabling that option in the settings will result in 404s.

### Solution Brainstorm

Could we update the default patterns to also match requests like `*/ping/` and `*/health/`, etc.?

### Product Area

Ingestion and Filtering

### PR DESCRIPTION
**Pull request by jan-auer**
Detect health check routes with an without a trailing slash.

Fixes https://github.com/getsentry/sentry/issues/81028

### CODE REVIEW

**Review by loewenheim — APPROVED** [review]

File: src/sentry/constants.py [hunk 1]
```diff
@@ -790,6 +790,19 @@ class InsightModules(Enum):
...
+    "*/healthy/",
+    "*/healthz/",
+    "*/_health/",
+    r"*/\[_health\]/",
+    "*/live/",
+    "*/livez/",
+    "*/ready/",
+    "*/readyz/",
+    "*/ping/",
+    "*/up/",
```
Comment (Dav1dde) [inline src/sentry/constants.py:805]: Should we do this in Code instead? Automatically duplicate all entries with a slash?

**Review by Dav1dde — APPROVED** [review]

File: src/sentry/constants.py (hunk 1 above)
Comment (jan-auer) [reply src/sentry/constants.py:805]: Will wait for https://github.com/getsentry/relay/pull/4277 instead.

Comment (loewenheim) [reply src/sentry/constants.py:805]: That PR has been superseded by https://github.com/getsentry/relay/pull/4283, which is now merged.

File: tests/sentry/relay/snapshots/test_config/test_get_project_config/REGION.pysnap [hunk 2]
```diff
@@ -94,6 +94,17 @@ config:
...
+      - '*/healthy/'
+      - '*/healthz/'
+      - '*/_health/'
+      - '*/\[_health\]/'
+      - '*/live/'
+      - '*/livez/'
+      - '*/ready/'
+      - '*/readyz/'
+      - '*/ping/'
+      - '*/up/'
```
Comment (loewenheim) [inline tests/sentry/relay/snapshots/test_config/test_get_project_config/REGION.pysnap:107]: Are the routes still supposed to be duplicated in this file?

**Review by loewenheim — APPROVED** [review]

**Review by Dav1dde — APPROVED** [review]

**Pull request merged**